Makefile

```
@@ -1,34 +1,48 @@
+		echo -e "$(ERROR_COLOR)Environment variable $* not set$(NO_COLOR)"; \
+		exit 1; \
+	fi
+
+check-%:
+	@if $$(hash $* 2> /dev/null); then \
+		echo -e "$(OK_COLOR)$(OK)$(NO_COLOR) $*"; \
+	else \
+		echo -e "$(ERROR_COLOR)$(KO)$(NO_COLOR) $*"; \
+	fi
+
+.PHONY: clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)